Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -0,0 +1,136 @@
+            memoryReader.Dispose();
+        }
+
+        //https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk65143
+        public void Collect(HostMetrics metrics)
+        {
+            var systemStat = ReadSystemStat();
+            var memInfo = ReadMemoryInfo();
+
+            if (systemStat.Filled)
+            {
+                var systemTime = systemStat.UserTime.Value + systemStat.NicedTime.Value + systemStat.SystemTime.Value +
+                                 systemStat.IOWaitTime.Value + systemStat.InterruptsTime.Value +
+                                 systemStat.SoftInterruptsTime.Value;
+
+                cpuCollector.Collect(metrics, systemTime, systemStat.IdleTime.Value);
+            }
+
+            if (memInfo.Filled)
+            {
+                metrics.MemoryAvailable = memInfo.AvailableMemory.Value;
+                metrics.MemoryCached = memInfo.CacheMemory.Value;
+                metrics.MemoryKernel = memInfo.KernelMemory.Value;
+                metrics.MemoryTotal = memInfo.TotalMemory.Value;
+            }
+        }
+
+        private SystemStat ReadSystemStat()
+        {
+            var result = new SystemStat();
+
+            try
+            {
+                if (FileParser.TrySplitLine(systemStatReader.ReadFirstLine(), 7, out var parts) && parts[0] == "cpu")
+                {
+                    if (ulong.TryParse(parts[1], out var utime))
+                        result.UserTime = utime;
+
+                    if (ulong.TryParse(parts[2], out var ntime))
+                        result.NicedTime = ntime;
+
+                    if (ulong.TryParse(parts[3], out var stime))
+                        result.SystemTime = stime;
+
+                    if (ulong.TryParse(parts[4], out var itime))
+                        result.IdleTime = itime;
+
+                    if (ulong.TryParse(parts[5], out var iotime))
+                        result.IOWaitTime = iotime;
+
+                    if (ulong.TryParse(parts[6], out var intime))
+                        result.InterruptsTime = intime;
+
+                    if (ulong.TryParse(parts[7], out var sintime))
+                        result.SoftInterruptsTime = sintime;
+                }
+            }
+            catch (Exception error)
+            {
+                InternalErrorLogger.Warn(error);
+            }
+
+            return result;
+        }
+
+        private MemoryInfo ReadMemoryInfo()
+        {
+            var result = new MemoryInfo();
+
+            try
+            {
+                foreach (var line in memoryReader.ReadLines())
+                {
+                    if (FileParser.TryParseLong(line, "MemTotal", out var memTotal))
+                        result.TotalMemory = memTotal * 1024;
+
+                    if (FileParser.TryParseLong(line, "MemAvailable", out var memAvailable))
+                        result.AvailableMemory = memAvailable;
```

Comment:
Why do you only multiply total memory by 1024? From what I gather, all values measured here are expressed in kilobytes in `/proc/meminfo`.